modules/Bio/EnsEMBL/Analysis/Hive/Config/long_read.pm

```diff
@@ -0,0 +1,589 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::long_read;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Analysis::Hive::Config::HiveBaseConfig_conf');
+
+sub default_options {
+  my ($self) = @_;
+  return {
+
+    # inherit other stuff from the base class
+    %{ $self->SUPER::default_options() },
+
+    ######################################################
+    #
+    # Variable settings- You change these!!!
+    #
+    ######################################################
+    ########################
+    # Misc setup info
+    ########################
+    'dbowner' => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name' => '' || $self->o('production_name') . '_' . $self->o('ensembl_release'),
+    'user_r'           => '',    # read only db user
+    'user'             => '',    # write db user
+    'password'         => '',    # password for write db user
+    'pipe_db_server'   => '',    # host for pipe db
+    'databases_server' => '',    # host for general output dbs
+    'dna_db_server'    => '',    # host for dna db
+    'databases_port'   => '',    # port for general output db host
+
+    'long_read_summary_file'       => '' || catfile( $self->o('long_read_dir'), $self->o('species_name') . '_long_read.csv' ),        # csv file for minimap2, should have 2 columns tab separated cols: sample_name\tfile_name
+    'long_read_summary_file_genus' => '' || catfile( $self->o('long_read_dir'), $self->o('species_name') . '_long_read_gen.csv' ),    # csv file for minimap2, should have 2 columns tab separated cols: sample_name\tfile_name
+    'long_read_fastq_dir' => '' || catdir( $self->o('long_read_dir'), 'input' ),
+
+    'release_number' => '' || $self->o('ensembl_release'),
+    'species_name'    => '',                                                                                                          # e.g. mus_musculus
+    'production_name' => '',                                                                                                          # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'output_path'     => '',                                                                                                          # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'skip_long_read'  => '0',                                                                                                         # Will skip long read analyses if 1
```

Comment:
This can be removed, it will be used in the main pipeline config

Reply:
'species_name' is referenced in a few places in the pipeline, but let me know in case it should really be deleted. (I guess 'species_name' is the only value you are referring to here. It is possible to scroll-select a single line if that's the case.)

Reply:
Hmmm when I scroll select 1 line it usually shows four to people. And it shows the correct lines when selecting more than one line. The line I select is the closest to the comment.
It's `skip_long_read` that can be removed

Reply:
Oh yes, that's superfluous for sure. I see what you mean when selecting a single line, I'll search again in case there is a way to do this.